inc/Engine/Optimization/CSSTrait.php

```diff
@@ -207,11 +207,14 @@ private function apply_font_display_swap( $css_file_content ) {
 			'/(?:@font-face)\s*{(?<value>[^}]+)}/',
 			function ( $matches ) {
 				if ( false !== strpos( $matches['value'], 'font-display' ) ) {
-					return $matches[0];
+					preg_match( '/font-display:\s*(?<attrib>\w*);?/', $matches['value'], $attribute );
```

Comment:
Why searching twice, one with strpos and another one with regex?
what about using only one check
```suggestion
					if ( preg_match( '/font-display:\s*(?<attrib>\w*);?/', $matches['value'], $attribute ) ) {
```
also please use regex `i` modifier (case insensitive).